src/app/stores/wallet/actions/actionCreators.ts

```diff
@@ -235,6 +235,15 @@ export function ChangeShowOrdinalReceiveAlertAction(
   };
 }
 
+export function ChangeShowDataCollectionAlertAction(
```

Comment:
```suggestion
export function changeShowDataCollectionAlertAction(
```

Reply:
I agree with your suggestion, I think we should change all the action names to camelCase with a small first letter as currently we have a mixed approach:
<img width="875" alt="Monosnap actionCreators ts — xverse-web-extension 2023-09-14 18-59-31" src="https://github.com/secretkeylabs/xverse-web-extension/assets/36603049/a2533d79-84c9-43f7-9ecb-832851ca5325">